plumgrid_playbooks/roles/plumgrid_neutron/tasks/main.yml

```diff
@@ -52,10 +32,15 @@
     file: /tmp/GPG-KEY
     state: present
 
-# Update repositories
-- name: Running apt-update
-  apt:
-    update_cache: yes
+- name: Create PLUMgrid sources.list
+  apt_repository:
+    repo: "{{ item }}"
+    filename: plumgrid
+    state: present
+  with_items:
```

Comment:
Instead of adding one by one, use the `with_items` with the `apt_repository` module to add all the repos.